index.html

```diff
@@ -6,9 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
   <link rel="stylesheet" href="css/styles.css"/>
-  <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
```

Comment:
kindly make sure the content  of your all page has the max of 100% so that it will contain all screens and remove the scrolling button, check below the responsiveness of your page compare to the Figma design and please try to fix it 

## your design 

![image](https://user-images.githubusercontent.com/95297251/186995046-d4988f39-af6c-4497-9062-c6f7dd128cf7.png)

## Figma design
![image](https://user-images.githubusercontent.com/95297251/186995161-b626a506-0bb2-4b6a-bc2d-d4c106402c3b.png)